docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"
```

Comment:
3.9 raised an error in ubuntu 20.04 with docker engine 20.10.12
probably a bug 🤔